js-api-doc/importer.d.ts

```diff
@@ -308,6 +308,37 @@ export interface Importer<sync extends 'sync' | 'async' = 'sync' | 'async'> {
   nonCanonicalScheme?: string | string[];
 }
 
+/**
+ * An opaque branded type, representing the built-in Node Package Importer. This
+ * is used for type checking. Users should use the exposed `nodePackageImporter`
+ * and not instantiate their own.
+ *
+ * @category Importer
+ */
+export type NodePackageImporter = {
+  _NodePackageImporterBrand: unknown;
+};
+/**
+ * The built-in Node Package Importer, which is a Package Importer that resolves
+ * using the standards and conventions of the Node ecosystem. It enables a
+ * `pkg:` URL scheme for usage with `@use` that directs an implementation to
+ * resolve a URL within a dependency.
```

Comment:
This will be the primary documentation that people will look to for information on this importer, so it's worth being particularly clear and thorough. A few specifics:

* "Node" should probably be "Node.js" throughout, since that's the full name of the platform.
* I don't think "Package Importer" should be capitalized—it's not a proper noun. That said, we _should_ find a place to document what it means to be a "package importer" and link to it here.
* "resolves using" isn't really proper grammar.
* "Enables a `pkg:` URL scheme" is pretty vague.
* "directs an implementation" is very confusing. Documentation inherently describes the behavior of an implementation—in what sense does directing an implementation to do something differ from just doing that thing?

I'd rewrite it as something like:

> The built-in Node.js [package importer](https://example.org). This loads `pkg:` URLs from `node_modules` according to the standard Node.js resolution algorithm.

You should also explicitly describe how to make a package compatible with it, all the ways it handles exports and style declarations in `package.json`, and so on. Remember, this is _the only place_ that a reader will be able to see this information—no one is going to go look up the proposal once this is released.

Reply:
Updated in 9d1d3d2c4d9f04f9a3d4021cf06ee93fe089e531